lib/features/create/Create.js

```diff
@@ -55,6 +55,11 @@ export default function Create(
       return !element.parent;
     });
 
+    // ignore labels being created with their targets
+    elements = filter(elements, function(element) {
+      return !(isLabel(element) && elements.indexOf(element) > -1);
```

Comment:
Is it possible that `elements.indexOf(element) > -1` is false in this case?

Reply:
Ah yes, this should be element.labelTarget.